feconf.py

```diff
@@ -520,6 +520,12 @@ def get_empty_ratings():
 UPDATE_TYPE_COLLECTION_COMMIT = 'collection_commit'
 UPDATE_TYPE_FEEDBACK_MESSAGE = 'feedback_thread'
 
+# User query status.
```

Comment:
Ah -- you used to have something like "the possible transitions are completed --> archived" etc. I meant that.